libs/nft/src/extensions/burnable/burnable.sw

```diff
@@ -26,8 +26,8 @@ impl Burnable for NFTCore {
     fn burn(self) {
         require(self.owner == msg_sender().unwrap(), AccessError::SenderNotOwner);
 
-        store(sha256((BALANCES, self.owner)), get::<u64>(sha256((BALANCES, self.owner))) - 1);
-        store(sha256((TOKENS, self.token_id)), Option::None::<NFTCore>());
+        store(sha256((BALANCES, self.owner)), get::<u64>(sha256((BALANCES, self.owner))).unwrap() - 1);
```

Comment:
```suggestion
        store(sha256((BALANCES, self.owner)), get::<u64>(sha256((BALANCES, self.owner))).unwrap_or(0) - 1);
```
thoughts?

Reply:
0 - 1 would revert due to a negative number. This value shouldn't ever be zero if we reach this far into the `burn()` function due to the revert case above.